basic_hashtable/b_hashtables.c

```diff
@@ -94,7 +109,17 @@ void hash_table_insert(BasicHashTable *ht, char *key, char *value)
  ****/
 void hash_table_remove(BasicHashTable *ht, char *key)
 {
+  unsigned int index = hash(key, ht->capacity);
 
+  if (ht->storage[index] == NULL)
+  {
+    printf("Nonexistent key: %s", key);
+    exit(1);
```

Comment:
in "real life" this program would be a tool for something else and it stopping all execution with an `exit(1)` won't be the desired behavior.